api/insight/converter.go

```diff
@@ -112,7 +110,7 @@ func (c *insightApiContext) TxConverterWithParams(txs []*dcrjson.TxRawResult, no
 		txNew.Fees = dcramt.ToCoin()
 
 		// Return true if coinbase value is not empty, return 0 at some fields
-		if txNew.Vins != nil && len(txNew.Vins[0].CoinBase) > 0 {
+		if txNew.Vins != nil && txNew.Vins[0].CoinBase != "" {
```

Comment:
either way is valid, but for strings, the semantics are clear from the syntax on the right without even knowing the type of `CoinBase`